.github/workflows/sync-release-branch-2023a.yml

```diff
@@ -7,27 +7,31 @@ on:
    - cron: "0 13 * * 6"
```

Comment:
May I ask you to change this cron to `0 8 * * 2` ? (Since we have this PR open :) )

Reply:
sure, if `0 8 * * 2` works better than old cronjob